frontend/src/metabase/dashboard/components/DashboardSidebars.jsx

```diff
@@ -92,12 +92,7 @@ export function DashboardSidebars({
 
   switch (sidebar.name) {
     case SIDEBAR_NAME.addQuestion:
-      return (
-        <AddCardSidebar
-          initialCollection={dashboard.collection_id}
```

Comment:
Remove the dependency on this prop because I need to access the `dashboard` in `frontend/src/metabase/dashboard/components/add-card-sidebar/QuestionPicker.jsx` anyway.